src/App/Osu/Osu.cpp

```diff
@@ -65,7 +65,7 @@
 
 // release configuration
 bool Osu::autoUpdater = false;
-ConVar osu_version("osu_version", 33.05f);
+ConVar osu_version("osu_version", 33.06f);
```

Comment:
The version number is something only I update, not what you would include in a pr